src/TimeService.Models.Dto/Models/WorkTimeMonthLimitInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace LT.DigitalOffice.TimeService.Models.Dto.Models
+{
+    public record WorkTimeMonthLimitInfo
+    {
+        public Guid Id { get; set; }
+        public int Month { get; set; }
+        public int Year { get; set; }
+        public float NormHours { get; set; }
```

Comment:
What is this property?

Reply:
count of working hours

Reply:
may be that named WorkingHours?

Reply:
The working time sounds like it's the time I've already worked